Solutions/Illusive Active Defense/Analytic Rules/Illusive_Detection_Query.yaml

```diff
@@ -0,0 +1,46 @@
+id: 1a7dbcf6-21a2-4255-84b2-c8dbbdca4630
+name: Illusive Incidents Analytic Rule
+description: |
+  'Create a Sentinel incident upon a new Illusive alert (incident) and associate all related Illusive events to the relevant Sentinel alert. This is done by filtering and processing Illusive Syslog messages.'
+severity: Medium
+requiredDataConnectors: 
+  - connectorId: Illusive
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+relevantTechniques:
+query: |
+  CommonSecurityLog
+  | where DeviceProduct == "illusive"
+  | summarize arg_max(TimeGenerated, *) by DeviceCustomNumber2, AdditionalExtensions, TimeGenerated
+  | extend Category = extract(@'cat=([^;]+)(\;|$)', 1, AdditionalExtensions), HasForensics = extract(@'cs7=([^;]+)(\;|$)', 1, AdditionalExtensions)
+  | where Category == "illusive:alerts"
```

Comment:
Suggest to format output to show key field for better analyst experience.. here is an example to add.

```
| extend IncidentId = DeviceCustomNumber2, IncidentURL = DeviceCustomString5
| project TimeGenerated, DeviceEventClassID, Activity, DeviceAddress, SourceHostName, DestinationHostName, SourceUserName, DestinationUserName, IncidentId, IncidentURL

```